extensions/wikia/UserLogin/js/UserBaseAjaxForm.js

```diff
@@ -62,19 +75,34 @@
 
 	/**
 	 * Make the call to the back end to log the user in via ajax
+	 * @abstract
 	 */
-	UserBaseAjaxForm.prototype.ajaxLogin = function () {
-		$.nirvana.postJson(
-			'UserLoginSpecial',
-			'login',
-			{
-				loginToken: this.loginToken,
-				username: this.inputs.username.val(),
-				password: this.inputs.password.val(),
-				keeploggedin: this.inputs.keeploggedin.is(':checked')
-			},
-			this.submitLoginHandler.bind(this)
-		);
+	UserBaseAjaxForm.prototype.ajaxLogin = function () {};
```

Comment:
The idea behind this is that it's an abstract class and method, even though JavaScript doesn't really have that concept, so it looks like it's just an empty function. Since all child classes override this method, there's no point in putting anything in here. However, it is a required method, so I don't want to delete it. 

I didn't see any empty functions in UserLogin.js. Are you sure that's the file you meant?